_config.yml

```diff
@@ -9,10 +9,11 @@ kramdown:
   syntax_highlighter: rouge
 highlights: true
 highlighter: rouge
+timezone: UTC # To match it with Circle CI default settings
 gems:
   - jekyll-redirect-from
   - jekyll-sitemap
-exclude: ['Gemfile*', 'vendor', 'README.md', 'Rakefile']
+exclude: ['Gemfile*', 'vendor', 'README.md', 'Rakefile', 'CNAME']
```

Comment:
Hmm, I thought having it in master is enough - https://help.github.com/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site/ can keep it there just to be sure